js/main.js

```diff
@@ -1,6 +1,7 @@
 // header import
 import headerEffect from "./headerJS.js";
 import sandwich from "./heroSandwich.js"
+import HeroScroll from "./scrollFromHero.js"
```

Comment:
neturetu buti is mazosios heroScroll? cia tik del bendros tvarkos :) (ir visur kitur pakeisti)